README.md

```diff
@@ -59,6 +59,10 @@ $ swift package format --exclude Tests
 # Alternatively you can explicitly list the set of paths and/or SPM targets:
 $ swift package format --paths Sources Tests Package.swift
 $ swift package format --targets AirbnbSwiftFormatTool
+
+# The plugin infers your package's minimum Swift version from the `swift-tools-version`
+# in your `Package.swift`, but you can provide a custom value with `--swift-version`:
+$ swift package format --swift-version 5.3
```

Comment:
Are we sure this works? In our code above we are _always_ creating `--swift-version` option and passing that to the underlying tool. If someone executed this command and their Package.swift had a tools version of 5.6 I think we'd pass `--swift-version 5.3 --swift-version 5.6` to the tool.

Reply:
It seems like we should avoid constructing a default option for the underlying tool if the user specified `--swift-version` manually.

Reply:
This command behaves as expected. I suppose the plugin is passing e.g. `--swift-version 5.6 --swift-version 5.3` to the underlying tool, and since this option isn't array Swift Argument Parser is selecting the last value. That sounds like its potentially relying on UB, so I agree that we should avoid passing default option if the user also provides one. Will post a PR.